static/admin-widget-style.css

```diff
@@ -7,29 +23,25 @@
     margin-bottom: 10px;
 }
 
-/* Clear floats after the columns */
-.row {
-    display: flex;
-}
-
-/* Layout styles*/
 .smaily-layouts {
     width: 90%;
 }
 
 /* Button styles */
-.column {
-    flex: 50%;  
+.row > p:first-child {
+    margin-bottom: 0.5em;
+}
+.row > p:last-child {
+    margin: 0.5em 1px 1em;
 }
 
 .button-style {
-    width: 95%;
-    border: 1px solid #c3c4c7;
+    width: 100%;
+	border: 1px solid #8c8f94;
```

Comment:
Formatting: indentation got a bit borked with the change.

Change it back to 4 spaces for now. In the future a maintenance PR is required to convert all of the indentation to tabs, as is required by the WordPress coding standard.